packages/nextjs/src/client/index.ts

```diff
@@ -61,6 +62,16 @@ export function init(options: BrowserOptions): Client | undefined {
     addEventProcessor(devErrorSymbolicationEventProcessor);
   }
 
+  try {
+    // @ts-expect-error `process.turbopack` is a magic string that will be replaced by Next.js
+    if (process.turbopack) {
+      getGlobalScope().setTag('turbopack', true);
```

Comment:
hmm are we sure we want to set a tag over a context here? Is there value in making this searchable?

Reply:
I say yes - for now. Sigrid is working on some stuff in the protocol that will allow us to send stuff like this in a more structured way. It will be invaluable for us in the transitionary period when turbopack ships and we have to debug potential issues with the SDK.